independent-publisher-connectors/Placedog/readme.md

```diff
@@ -0,0 +1,23 @@
+# Placedog
+Request dogs from placedog.net. Hand-picked, stunning photographs of our favorite pets: dogs! Free to use with your websites/projects! Featured in the #ThePawryParty Chrome Extension!

```

Comment:
@natalie-pienkowska Can you please let me know if I can go ahead with this submission ?